benchmark/paddle/image/resnet.py

```diff
@@ -1,16 +1,3 @@
-#  Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
Why delete the copyright under directory `benchmark/paddle`

Reply:
will be added back to next PR.